modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/Actions.tsx

```diff
@@ -92,6 +93,67 @@ export default function Actions({
 		);
 	}
 
+	function objectActionSourceDataRenderer({itemData}: {itemData: ItemData}) {
+		return (
+			<strong
+				className={classNames(
+					itemData.system ? 'label-info' : 'label-warning',
+					'label'
+				)}
+			>
+				{itemData.system
+					? Liferay.Language.get('system')
+					: Liferay.Language.get('custom')}
+			</strong>
+		);
+	}
+
+	const fdsFields = [
```

Comment:
@seltong @G-Sa after moving the DataRenderers you can also declare this const outside the Actions component.